diff -ur -x configure tex-fusecompress-c5e5eb58e48a6ad08298f178a9d91b9539abf883.org/src/boost/archive/portable_binary_iarchive.hpp tex-fusecompress-c5e5eb58e48a6ad08298f178a9d91b9539abf883/src/boost/archive/portable_binary_iarchive.hpp
--- tex-fusecompress-c5e5eb58e48a6ad08298f178a9d91b9539abf883.org/src/boost/archive/portable_binary_iarchive.hpp	2008-11-28 11:08:35.000000000 +0100
+++ tex-fusecompress-c5e5eb58e48a6ad08298f178a9d91b9539abf883/src/boost/archive/portable_binary_iarchive.hpp	2008-11-28 14:24:28.293312642 +0100
@@ -46,8 +46,10 @@
 #include <boost/archive/archive_exception.hpp>
 #if BOOST_VERSION < 103500
 #include <boost/archive/detail/polymorphic_iarchive_impl.hpp>
-#else
+#elif BOOST_VERSION < 103600
 #include <boost/archive/detail/polymorphic_iarchive_dispatch.hpp>
+#else
+#include <boost/archive/detail/polymorphic_iarchive_route.hpp>
 #endif
 
 #include <boost/integer/endian.hpp>
@@ -145,8 +147,10 @@
 	friend primitive_base_t; // since with override load below
 #if BOOST_VERSION < 103500
 	friend class boost::archive::detail::polymorphic_iarchive_impl<derived_t>;
-#else
+#elif BOOST_VERSION < 103600
 	friend class boost::archive::detail::polymorphic_iarchive_dispatch<derived_t>;
+#else
+	friend class boost::archive::detail::polymorphic_iarchive_route<derived_t>;
 #endif
 	friend class boost::archive::basic_binary_iarchive<derived_t>;
 	friend class boost::archive::load_access;
@@ -273,7 +277,10 @@
 #if BOOST_VERSION < 103500
 typedef boost::archive::detail::polymorphic_iarchive_impl<
 	portable_binary_iarchive> polymorphic_portable_binary_iarchive;
-#else
+#elif BOOST_VERSION < 103600
 typedef boost::archive::detail::polymorphic_iarchive_dispatch<portable_binary_iarchive> 
 	polymorphic_portable_binary_iarchive;
+#else
+typedef boost::archive::detail::polymorphic_iarchive_route<portable_binary_iarchive>
+        polymorphic_portable_binary_iarchive;
 #endif
Tylko w tex-fusecompress-c5e5eb58e48a6ad08298f178a9d91b9539abf883/src/boost/archive: portable_binary_iarchive.hpp~
diff -ur -x configure tex-fusecompress-c5e5eb58e48a6ad08298f178a9d91b9539abf883.org/src/boost/archive/portable_binary_oarchive.hpp tex-fusecompress-c5e5eb58e48a6ad08298f178a9d91b9539abf883/src/boost/archive/portable_binary_oarchive.hpp
--- tex-fusecompress-c5e5eb58e48a6ad08298f178a9d91b9539abf883.org/src/boost/archive/portable_binary_oarchive.hpp	2008-11-28 11:08:35.000000000 +0100
+++ tex-fusecompress-c5e5eb58e48a6ad08298f178a9d91b9539abf883/src/boost/archive/portable_binary_oarchive.hpp	2008-11-28 14:24:00.259980684 +0100
@@ -45,8 +45,10 @@
 #include <boost/archive/binary_oarchive.hpp>
 #if BOOST_VERSION < 103500
 #include <boost/archive/detail/polymorphic_oarchive_impl.hpp>
-#else
+#elif BOOST_VERSION < 103600
 #include <boost/archive/detail/polymorphic_oarchive_dispatch.hpp>
+#else
+#include <boost/archive/detail/polymorphic_oarchive_route.hpp>
 #endif
 
 #include <boost/integer/endian.hpp>
@@ -110,8 +112,10 @@
 	friend primitive_base_t; // since we override save below
 #if BOOST_VERSION < 103500
 	friend class boost::archive::detail::polymorphic_oarchive_impl<derived_t>;
-#else
+#elif BOOST_VERSION < 103600
 	friend class boost::archive::detail::polymorphic_oarchive_dispatch<derived_t>;
+#else
+	friend class boost::archive::detail::polymorphic_oarchive_route<derived_t>;
 #endif
 	friend class boost::archive::basic_binary_oarchive<derived_t>;
 	friend class boost::archive::save_access;
@@ -261,7 +265,10 @@
 #if BOOST_VERSION < 103500
 typedef boost::archive::detail::polymorphic_oarchive_impl<
 	portable_binary_oarchive> polymorphic_portable_binary_oarchive;
-#else
+#elif BOOST_VERSION < 103600
 typedef boost::archive::detail::polymorphic_oarchive_dispatch<portable_binary_oarchive> 
 	polymorphic_portable_binary_oarchive;
+#else
+typedef boost::archive::detail::polymorphic_oarchive_route<portable_binary_oarchive>
+        polymorphic_portable_binary_oarchive;
 #endif

